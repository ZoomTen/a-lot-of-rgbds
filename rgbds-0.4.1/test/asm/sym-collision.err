ERROR: sym-collision.asm(26):
    'dork' not defined
error: Assembly aborted (1 errors)!
