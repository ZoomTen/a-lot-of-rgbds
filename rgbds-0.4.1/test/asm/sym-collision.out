aqfj: $FE00
cxje: $FE01
dgsd: $FE02
dork: $0
lxok: $FE04
psgp: $FE05
sfly: $FE06
syyq: $FE07
ussg: $FE08
xlmm: $FE09
xtzp: $FE0A
zxfr: $FE0B
